src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/statuses/payments/list/state_label.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set state = hookable_metadata.context.payment.state %}
+
+{% if state != 'cart' %}
+    <p {{ sylius_test_html_attribute('payment-state') }}>
+        {% include "@SyliusShop/shared/order/Label/payment_state/single_payment_state.html.twig" with { 'state': state } %}
```

Comment:
imo having one macro for all order labels would be better. And then we can use it everywhere instead includes